Vostok.Metrics.System/Host/DiskUsageInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿using JetBrains.Annotations;
+
+namespace Vostok.Metrics.System.Host
+{
+    [PublicAPI]
+    public class DiskUsageInfo
+    {
+        public string DiskName { get; set; }
+        public double IdleTimePercent { get; set; }
+        public double ReadLatency { get; set; }
+        public double WriteLatency { get; set; }
+        public double ReadsPerSecond { get; set; }
```

Comment:
Seems like I forgot to include two useful metrics in the initial issue description: bytes read/sec and bytes written/sec. Does it look like something you could add?